### ISSUE TITLE
Hostile Paradise? More like hostile experience

### ISSUE THREAD

**Issue by sicut-anima** [Gilded-Games/The-Aether-II#166]
First off, there's no auto-jump option. This requires me to keep tapping the space bar constantly while exploring, aggravating my repetitive strain injury. Secondly, the recipe book no longer exists. Thirdly, Nausea IV really is nauseating. After defeating only a couple of golems, I had to take a break before I got too sick to play. Is there an option to reduce the effects? Of course not! Why would I want that, when I have a "super-secret menu" that adds innumerable eye-straining filters, with no indication of what the button will do when I press it? Trying to turn the filters off only adds new ones.
tdlr Aether II is disabled hostile

**Comment by ZolaKluke** [comment]
1. Auto-jump and the recipe book didn't exist in 1.7.10
2. Super Secret Menu is an old Minecraft feature that yes contains horrible filters, it's not an Aether 2 thing at all it's a vanilla thing
3. Yeah the nausea golems are a pain, but unfortunately the 1.7.10 version hasn't been updated in like 6 years so that'll never be changed

TLDR: Try playing old versions of Minecraft before blaming the mod for most things

**Comment by sicut-anima** [comment]
> 1. Auto-jump and the recipe book didn't exist in 1.7.10
> 2. Super Secret Menu is an old Minecraft feature that yes contains horrible filters, it's not an Aether 2 thing at all it's a vanilla thing
> 3. Yeah the nausea golems are a pain, but unfortunately the 1.7.10 version hasn't been updated in like 6 years so that'll never be changed
> 
> TLDR: Try playing old versions of Minecraft before blaming the mod for most things

The Aether is one of those things where the sequel's way worse. Except for companions.

**Comment by bconlon1** [comment]
Tfw you get a bug report about Minecraft features.
